content/javascript/concepts/dates/terms/setUTCSeconds/setUTCSeconds.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.setUTCSecond()'
+Description: 'Sets the second value of a date object using UTC.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.setUTCSeconds()`** method sets the second value of a `Date` object according to Coordinated Universal Time(UTC) and returns the updated `Date` object.
+
+## Syntax
+
+```
+myDate.setUTCSeconds(secondsValue)
+myDate.setUTCSeconds(secondsValue, millisecondsValue)
+```
+The `.setUTCSeconds()` method is called on the `myDate` object with an input argument `secondsValue` that passes the new second value.
+
+Optionally, the method can also set the date's millisecond UTC values. The valid number ranges for the time units are as follows:
+
+- **Seconds** (Required): _0_ to _59_
+- **Milliseconds** (optional): _0_ to _999_
+
+## Examples
+
+This example sets the second value of the `eventSeconds` object to `35`.
+
+```javascript
+const eventSeconds = new Date('2023-06-25T00:00:00');
+d.setUTCSeconds(35);
```

Comment:
```suggestion
eventSeconds.setUTCSeconds(35);
```

Reply:
Additionally, I think it would be more complete to include a line such as:
console.log(eventSeconds.toString())

And include the output in a ```shell block